docs/sheets/GNSS.md

```diff
@@ -1,236 +1,251 @@
-> * :three: :pencil2: Update the __install/antenna.csv__ file to add or update any installed antennas, the entries will need to be in order of make, model, serial number, and installation time.
-> * :four: :pencil2: Edit the __install/receivers.csv__ file to add or update any deployed receivers, the entries will need to be in order of make, model, serial number, and deployment time.
-> * :five: :pencil2: Update the __install/firmware.csv__ file to add or adjust the deployed __receiver__ "firmware" versions, the entries will need to be in order of make, model, serial number, and firmware start times.
-> * :six: :pencil2: Update the __install/sessions.csv__ file to add or adjust the deployed __receiver__ __session__ settings, the entries will need to be in order of __mark__ "code", session "interval", and session start times.
-> * :open_file_folder: Build the pull request with a meaningful title.
-> * :link: Assign suitable tags and set reviewers.
-> * :repeat: If the tests fail, the above changes may need some iteration until they pass.
-> * :sos: If the tests are still failing, escalate as this may indicate some inconsistency within the network configuration.
-> * :ok: Once the tests have passed and the pull request reviewed, depending on policy, the pull request can be merged
+## :six: <a name="openclose"></a>_Creating updating or closing a GNSS site_
+
+For new marks a __code__ will need to be assigned by the appropriate mechanism.
+Use also [antenna](#antenna) [receiver](#receiver) [firmware](#firwmare) and [session](#session) checklists
+
+> ### :page_with_curl: Files to update
+> * __network/marks.csv__
+> * __network/monuments.csv__
+> * __environment/visibility.csv__
+> * __install/antenna.csv__
+> * __install/receivers.csv__
+> * __install/firmware.csv__
+> * __install/sessions.csv__
+
+>
+> ### :page_with_curl: Reference files
+> * __network/networks.csv__
+> * __assets/antennas.csv__
+> * __assets/receivers.csv__
+>
+> ### :information_source: General requirements
+> * The GNSS __mark__ "code".
+> * The "name" of the GNSS __mark__.
+> * The "code" for the __network__ that the __mark__ is a member of.
+> * Whether the __mark__ is an "_IGS_" site.
+> * Precise coordinates of the GNSS __mark__
+> * The "domes number" of the __monument__.
+>
+> ### :information_source: Field requirements
+> * The location of the __mark__ to create or update.
+> * The operational time range of the __mark__ (Start is the first data in the archive).
+> * The "mark type" of the __monument__.
+> * The ground relationship of the __monument__.
+> * The __monument__ "foundation type".
+> * The __monument__ "foundation depth".
+> * The date the __monument__ was established.
+> * The date the __monument__ was demolished, if applicable.
+>
+> ### :heavy_check_mark: Delta prerequisites
+> * There can only be one __mark__ for any given code.
```

Comment:
should this be clearer: for any given mark "code" (same reference style used previously)